### PR TITLE
Remove current_store_id method, replacing input with graphql input

### PR DESCRIPTION
**Pull request by andreievg**
closes https://github.com/openmsupply/remote-server/issues/740

Ended up quite a big PR, also had to update graphlq.schema

### CODE REVIEW

File: server/src/main.rs [hunk 1]
```diff
@@ -48,6 +48,7 @@ async fn main() -> std::io::Result<()> {
         // TODO: disable once frontend supports auth!
         debug_no_access_control: true,
     });
+    println!("{:#?}", settings.database);
```
Comment (clemens-msupply) [inline server/src/main.rs:51]: Debug output?

File: repository/src/database_settings.rs [hunk 2]
```diff
@@ -3,7 +3,7 @@ use serde;
 
 use crate::db_diesel::{DBBackendConnection, StorageConnectionManager};
 
-#[derive(serde::Deserialize)]
+#[derive(serde::Deserialize, Debug)]
```
Comment (clemens-msupply) [inline repository/src/database_settings.rs:6]: Needed when debug output is removed?

**Review by clemens-msupply — APPROVED** [review]

File: server/src/main.rs (hunk 1 above)
Comment (andreievg) [reply server/src/main.rs]: thanks beb61786b4464f8c6ee6228b22df678da9081adf

File: repository/src/database_settings.rs (hunk 2 above)
Comment (andreievg) [reply repository/src/database_settings.rs]: beb61786b4464f8c6ee6228b22df678da9081adf

**Pull request merged**